src/fl_draw.cxx

```diff
@@ -592,67 +592,21 @@ void fl_draw_check(Fl_Rect bb, Fl_Color col) {
 } // fl_draw_check()
 
 /**
-  Draw a potentially small, filled circle as a GUI element.
+  Draw a potentially small, filled circle using a given color.
 
-  This method draws a filled circle, using fl_pie() if the given diameter
-  \p d is larger than 6 pixels after scaling with the current screen
-  scaling factor (i.e. "physical" pixels).
+  This function draws using \param color a filled circle bounded by rectangle <tt>(x, y, d, d)</tt>.
+ 
+  This function is the same as <tt>fl_pie(x, y, d, d, 0, 360)</tt> except with some systems
+  that don't draw small circles well. In that situation, the circle diameter \p d is converted
+  from FLTK units to pixels and this function approximates a filled circle by drawing several
+  filled rectangles if the converted diameter is ≤ 6 pixels.
 
-  If the diameter is 6 or smaller after scaling it approximates a filled circle
-  by drawing several filled rectangles, depending on the size because fl_pie()
-  might not draw well on many systems for smaller sizes.
-
-  \param[in]  x0,y0   coordinates of top left of the bounding box
-  \param[in]  d       diameter == width and height of the bounding box
+  \param[in]  x,y   coordinates of top left of the bounding box
+  \param[in]  d       diameter == width and height of the bounding box in FLTK units
```

Comment:
Sorry, nitpicking: please align descriptions for better reading.

Reply:
Committed at c855d93.